### PR TITLE
lab-8 roflmyrlok conflicts-resolved

### PR DESCRIPTION
**Pull request by roflmyrlok**

### CODE REVIEW

File: lecture_notes/08 collections/1-array-cpp.cpp [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
   int last = ages[length - 1];
-
-
   printf("first: %d\n", first);
   printf("last: %d\n", last);
+  
+  ages[4] = 4;
+  printf("new element: %d\n", ages[4]);
+  printf("function elements: %d\n");
+  for(int i = 0; i < length; i++) {
```
Comment (tshemsedinov) [inline lecture_notes/08 collections/1-array-cpp.cpp:17]:
```suggestion
  for (int i = 0; i < length; i++) {
```

File: lecture_notes/08 collections/1-array-modyfied.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+//usage:
+const ages = [10, 12, 15, 15, 17, 18, 18, 19, 20];
+const first = ages[0];
+const last = ages[ages.length - 1];
+const arrayForNewElements = ages.reverse();
+arrayForNewElements.filter((element) => element > 16 || element === 12);
```
Comment (tshemsedinov) [inline lecture_notes/08 collections/1-array-modyfied.js:8]: Assign result (filtered) to identifier

File: lecture_notes/08 collections/1-list-modified.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+ages = [10, 12, 15, 15, 17, 18, 18, 19, 20]
+
+def listPopFirstNLast(ages_new):
+    ages = ages_new.copy()
+    for i in range(int(len(ages))//2):
```
Comment (tshemsedinov) [inline lecture_notes/08 collections/1-list-modified.py:5]: Remove int

File: lecture_notes/08 collections/1-list-modified.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        ages.pop(-1)
+        ages.reverse()
+    print(ages)
+    return
+
+def listElementsChanges(ages_new):
+    ages = ages_new.copy()
+    listOfElementsToAdd = [321,322]
+    ages.extend(listOfElementsToAdd)
+    ages.insert(4,"mamamia")
```
Comment (tshemsedinov) [inline lecture_notes/08 collections/1-list-modified.py:31]:
```suggestion
    listOfElementsToAdd = [321, 322]
    ages.extend(listOfElementsToAdd)
    ages.insert(4, "mamamia")
```

File: lecture_notes/08 collections/2-function-editet.cpp [hunk 5]
```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+#include <string>
+
+void sumarray(int length,int array1[]){
+    int first = array1[0];
+    int last = array1[length-1];
+    std::cout << "First: ";
+    printf("%d",first);
+    std::cout << " ";
+    std::cout << "Last: ";
```
Comment (tshemsedinov) [inline lecture_notes/08 collections/2-function-editet.cpp:10]:
```suggestion
    std::cout << "\nLast: ";
```

**Review by tshemsedinov — COMMENTED** [review]
- Fix indentation in all languages
- Respect linter rules, see tab `Checks`
- Improve naming

File: lecture_notes/08 collections/1-array-modyfied.js (hunk 2 above)
Comment (roflmyrlok) [reply lecture_notes/08 collections/1-array-modyfied.js]: done

**Pull request closed without merging**